source/Item.hx

```diff
@@ -0,0 +1,86 @@
+package;
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import haxe.Timer;
+
+
+/**
+ * ...
+ * @author Cameron Yuan
+ */
+class Item extends FlxSprite
+{
+	private var player_trace:Player;
```

Comment:
There are several times where the position of the player, as well as the way the player is facing needs to be known by the weapon in order to have any animations in the correct direction, angles, x/y positions in relation to the player. I modeled the tracking off of the Sentry Enemy. Do you have another suggestions to keep track of these?